src/contents/ui/ScrollingText.qml

```diff
@@ -23,7 +23,11 @@ Item {
     property int speed: 5;
     readonly property int duration: (25 * (11 - speed) + 25)* textAndSpacing.length;
 
+    property bool scrollingEnabled: true
+    property bool scrollResetOnPause: false
+
     readonly property bool pauseScrolling: {
+        if (!scrollingEnabled) return true
```

Comment:
setting `paused` to True when the animation is not running cause this warning:
```sh
QML NumberAnimation: setPaused() cannot be used when animation isn't running
```

I think it's better to set animation `running` to False when is `scrollingEnabled` is False, something like:
```qml
running: root.overflow && root.scrollingEnabled
```

Reply:
Yeah, before this parch I see that warning too when changing songs, turns out the paused property needs to check if it's running, and also the reset function. With that there is no warnings.

```qml
paused: root.pauseScrolling && running
```

```qml
function reset() {
      label.x = 0;
      if (running) {
          restart()
      }
      if (running && root.pauseScrolling) {
          pause()
      }
  }
```